src/overlays/actors/ovl_En_Baba/z_en_baba.c

```diff
@@ -107,19 +128,19 @@ static DamageTable sDamageTable = {
     /* Powder Keg     */ DMG_ENTRY(1, 0x0),
 };
 
-static u8 D_80BAA488[] = {
+static u8 sSchedule[] = {
     /* 0x00 */ SCHEDULE_CMD_CHECK_NOT_IN_DAY_S(1, 0x1D - 0x04),
     /* 0x04 */ SCHEDULE_CMD_CHECK_NOT_IN_SCENE_S(SCENE_BACKTOWN, 0x1C - 0x08),
     /* 0x08 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(0, 0, 0, 30, 0x16 - 0x0E),
     /* 0x0E */ SCHEDULE_CMD_CHECK_BEFORE_TIME_S(0, 30, 0x15 - 0x12),
-    /* 0x12 */ SCHEDULE_CMD_RET_VAL_L(1),
+    /* 0x12 */ SCHEDULE_CMD_RET_VAL_L(BABA_SCHEDULE_RESULT_KNOCKED_OVER),
     /* 0x15 */ SCHEDULE_CMD_RET_NONE(),
-    /* 0x16 */ SCHEDULE_CMD_RET_TIME(0, 0, 0, 30, 2),
+    /* 0x16 */ SCHEDULE_CMD_RET_TIME(0, 0, 0, 30, BABA_SCHEDULE_RESULT_FOLLOW_TIME_PATH),
     /* 0x1C */ SCHEDULE_CMD_RET_NONE(),
     /* 0x1D */ SCHEDULE_CMD_RET_NONE(),
 };
 
-s32 D_80BAA4A8[] = { -1, -1, 0 };
+static s32 sAdditionalTimePaths[] = { -1, -1, 0 };
```

Comment:
```suggestion
static s32 sSearchTimePathLimit[] = {
    -1, // Get the last pathIndex in the path's connected by nextPathIndex (unused)
    -1, // Get the last pathIndex in the path's connected by nextPathIndex (unused)
    0,  // Get the first pathIndex in the path's connected by nextPathIndex
};
```
What about something like this? These are used as the last argument in `SubS_GetAdditionalPath(` which looking at that function (sorry for not reviewing those before, these path functions really interest me), That argument is the number of paths to search through. By setting that to 0, it means get the first path in the chained path list connected by unk1/nextPathIndex. But by setting that to 0, that means there is no limit and it grabs the last path in the chained path list connected by unk1/nextPathIndex

Reply:
I like the name change, but not sure about the comments. Some actors have quite a few so it seems silly to specify it would be the `first`, `second`, etc path.

Reply:
Note I also like the idea of changing that func's arg to limit instead of max.

Reply:
Yeah that's fine, it was more for my own understanding, and I'm pretty sure the docs I have above is wrong anyway taking a second look, because the first check in `SubS_GetAdditionalPath(`:
`i >= limit`
will always return true for the first path when `i = 0` and either `limit = 0` or `limit = -1`, so actually I'm not quite sure the difference in using `0` or `-1`?

Reply:
the `-1` seems to only have meaning in the actor itself. Like the bodyParts arrays where `-1` just means none.
`if (sSearchTimePathLimit[scheduleResult->result] >= 0)`